prusti-tests/tests/verify_overflow/fail/prusti_assert.rs

```diff
@@ -0,0 +1,21 @@
+#![allow(unused)]
+
+use prusti_contracts::*;
+
+fn assert1() {
+    prusti_assert!(true);
+}
+
+fn failing_assert() {
+    prusti_assert!(false); //~ ERROR
+    body_invariant!(false);
+}
+
+fn loop_shouldnt_crash() {
+    loop {
+        prusti_assert!(true);
+        body_invariant!(true);
+    }
+}
+
```

Comment:
Add tests with quantifiers and implications.

Reply:
Since lowering of quantifiers is not yet supported, let's land it as it is.